internal/git/utils.go

```diff
@@ -30,7 +30,7 @@ import (
 	"net/url"
 	"os"
 	"strings"
-
+	"slices"
```

Comment:
uplift is written using Go 1.19. I believe this package was introduced within Go 1.21. You will need to replace `slices.Contains` with an expanded `for` statement to replicate the same behaviour